app/views/notifier/order_receipt.html.haml

```diff
@@ -31,19 +32,23 @@
         %td= od
         - if od.reservation
           %td
-            = ((od.bundle ? "#{h od.bundle} &mdash; " : '') + h(od.product)).html_safe
+            = ((od.bundle ? "#{od.bundle} &mdash; " : "") + h(od.product)).html_safe
```

Comment:
There's a method in ApplicationHelper, `order_detail_description` which seems to have the same goal. We might be able to reuse that (although it could probably use a little cleanup as well).